docs/add-ons/promtail.md

```diff
@@ -0,0 +1,39 @@
+# PROMTAIL
+
+Promtail is an agent which ships the contents of local logs to a Loki instance.
+
+[PROMTAIL](https://github.com/grafana/helm-charts/tree/main/charts/promtail) chart bootstraps Promtail infrastructure on a Kubernetes cluster using the Helm package manager.
+
+For complete project documentation, please visit the [Promtail documentation site](https://grafana.com/docs/loki/latest/clients/promtail/).
+
+## Usage
+
+PROMTAIL can be deployed by enabling the add-on via the following.
```

Comment:
```suggestion
PROMTAIL can be deployed by enabling the add-on via the following.
```

Not sure why these are all caps but lets just use title case on all of these `Promtail`